app/pages/_app.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
```

Comment:
On mobile version you cant see the content for the info icon (looks like its only set to show on hover but mobile would need it to show on click as well)

https://user-images.githubusercontent.com/18233774/220048331-c5d7f32e-75ff-4c8d-b70d-972ba8ed3c7a.mov